tests/src/env.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(dead_code, unused)]
```

Comment:
This is the reason I was thinking of using ignore instead of `#[cfg]` straight up

Reply:
thank you! i understand your point now (: fixed!